dali/c_api/c_api.cc

```diff
@@ -105,20 +105,32 @@ void daliCreatePipeline(daliPipelineHandle* pipe_handle,
     pipe->SetQueueSizes(cpu_prefetch_queue_depth, gpu_prefetch_queue_depth);
   }
   pipe->Build();
-  pipe_handle->pipe = reinterpret_cast<void*>(pipe);
+  pipe_handle->pipe = reinterpret_cast<void *>(pipe);
```

Comment:
```suggestion
  pipe_handle->pipe = pipe;
```
Implicit cast to void* is allowed.